google-cloud-clients/google-cloud-storage/src/test/java/com/google/cloud/storage/BlobTest.java

```diff
@@ -71,11 +71,13 @@
   private static final String CONTENT_ENCODING = "UTF-8";
   private static final String CONTENT_LANGUAGE = "En";
   private static final String CRC32 = "0xFF00";
```

Comment:
I understand the confusion now. Sorry I didn't see this before, thanks for linking to it.

The hex values used for `CRC32` and `MD5` are not valid and only used for unit tests. I recommend replacing these values out with the base64 encoded crc32c and md5 values.